setup.py

```diff
@@ -135,29 +136,14 @@
     "bs4",
     "conllu",
     "h5py",
-    "langdetect",
     "lxml",
-    "lz4",
     "mwparserfromhell",
-    "nltk",
     "openpyxl",
     "py7zr",
-    "tldextract",
     "zstandard",
     "bigbench @ https://storage.googleapis.com/public_research_data/bigbench/bigbench-0.0.1.tar.gz",
     "sentencepiece",  # bigbench requires t5 which requires seqio which requires sentencepiece
-    "sacremoses",
-    # metrics dependencies
-    "bert_score>=0.3.6",
-    "jiwer",
-    "mauve-text",
-    "rouge_score<0.0.7",
-    "sacrebleu",
-    "sacremoses",
-    "scikit-learn",
-    "scipy",
-    "sentencepiece",  # for bleurt
-    "seqeval",
```

Comment:
I would keep the metric tests, we're not dropping them completly yet ;)

Reply:
@lhoestq but once metrics are officially deprecated, we are no longer maintaining them: IMO no enhancement and no bug fix should be done, and thus is why we recommend people switching to `evaluate`. Thus, testing makes no sense. The metrics are now fully supported, maintained and **tested** in `evaluate`.

Moreover, our CI will have a significant boost on performance once we remove all metrics test dependencies and tests.

Reply:
Metrics are connected to other parts of the codebase, and we don't want to break metrics if we change other parts of the codebase. I agree this is very unlikely, but we really don't want that to happen until we completely dropped the metrics. Note that there is much more usage of `load_metric` than `load_dataset` so we still want to be extra careful not to break them

Reply:
OK, I'm reverting the removal of metrics CI testing... :sob: 

I also removed it because I thought you agreed: in this PR description I asked:
> Maybe we should also stop testing metrics?
and juts your message afterwards was:
> Sounds good to me !

Reply:
Ah sorry it was confusing ^^'